src/azure-cli/azure/cli/command_modules/vm/_image_builder.py

```diff
@@ -377,17 +377,19 @@ def process_img_tmpl_output_add_namespace(cmd, namespace):
 
 # region Custom Commands
 
-def create_image_template(  # pylint: disable=too-many-locals, too-many-branches, too-many-statements
+def create_image_template(  # pylint: disable=too-many-locals, too-many-branches, too-many-statements, unused-argument
```

Comment:
Which argument is not used ? Can we remove the argument directly ?

Reply:
I didn't add unused argument. Maybe open another PR to address it.

Reply:
They are used in validator `process_image_template_create_namespace`